assignments/this.js

```diff
@@ -1,26 +1,54 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window/Global Object Binding - if "this" is used while in global scope, it will bind to the window object;
+* 2. Implicit Binding -used when referring to a property within an object, the word this followed by .
```

Comment:
Careful, the binding is done when a method is called off of an object (like `pebbles.speak()`). Then if `this` occurs in the function, it will be bound to `pebbles`